src/components/Jam.jsx

```diff
@@ -187,7 +185,8 @@ export default function Jam() {
 
     const body = { destination_addresses: destinations }
 
-    if (process.env.NODE_ENV === 'development' && useInsecureTestingSettings && !deactivateTestingToggle) {
+    // Make sure schedule testing is really only used in dev mode.
```

Comment:
```suggestion
```

What do you think about making the "insecure testing settings" a feature flag that is enabled by default on `env := 'development'` instead? I feel like this is more appropriate and would do away with the "double check", which was very surprising to me. Also, it _should in theory_ be possible to run Jam production build with a regtest backend, right?

Reply:
I had that initially but I wasn't sure if I liked it so I removed it again. My thinking was that this isn't a feature that will ever be turned on. It's more of a conditional thing that will always be on in dev and never in prod. 

What about adding a second "thing" to the feature flags code called "debug features" or something?

Reply:
> I had that initially but I wasn't sure if I liked it so I removed it again. My thinking was that this isn't a feature that will ever be turned on. It's more of a conditional thing that will always be on in dev and never in prod.

That's a good argument. I understand :+1:

> What about adding a second "thing" to the feature flags code called "debug features" or something?

Nah, let's see if it comes up again. I dont want to create more work artificially. If something like this comes up again, we can consider doing this.